csrc/transformer/inference/csrc/relu.cu

```diff
@@ -14,25 +16,21 @@ __global__ void fused_bias_relu(float* input,
                                 int total_count,
                                 int intermediate_size)
 {
-    float4* input_cast = reinterpret_cast<float4*>(input);
-    const float4* bias_cast = reinterpret_cast<const float4*>(bias);
-    int offset = blockIdx.x * blockDim.x + threadIdx.x;
+    // Input restriction: intermediate_size % vals_per_access == 0
+    constexpr int granularity = 16;
+    constexpr int vals_per_access = granularity / sizeof(float);
+    const int offset = (blockIdx.x * blockDim.x + threadIdx.x) * vals_per_access;
 
     if (offset < total_count) {
-        float4 data = input_cast[offset];
-        float4 bias_data = bias_cast[offset % intermediate_size];
+        float data[vals_per_access];
+        float data_bias[vals_per_access];
```

Comment:
@awan-10  Just following the same convention as gelu.cu

Reply:
@cmikeh2 -- okay so then this is a question for you. Is there a technical reason for calling this data_bias and not bias_data?

Reply:
It wasn't a conscious decision to change the variable name there to be totally honest. As long as it's descriptive that it's the bias data being loaded I'm happy with whatever we want to go with.

Reply:
Sounds good. Yeah, I only got hooked because its a diff and shows a rename. No problem if it was just a side effect. Resolving and merging the PR when tests pass.